src/libcore/tests/nonzero.rs

```diff
@@ -121,3 +122,13 @@ fn test_match_nonzero_const_pattern() {
         _ => panic!("Expected the const item as a pattern to match.")
     }
 }
+
+#[test]
+fn test_nonzero_const_fn_get() {
+    const nonzero_n: NonZeroU32 = NonZeroU32::new_unchecked(1);
```

Comment:
please make it two ui tests, one that tests whether not having the feature gate forbids using in constants dn one that has the feature gate and compiles successfully (`// compile-pass` as a comment in the file will allow that)

Reply:
Where are these "ui tests"?

Reply:
src/test/ui/const/